spec/models/daily_usage_creation/steps/appliance_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "pry"
```

Comment:
💅 Do you still need pry here?

Reply:
Naurrrr  - I've removed it from the spec files and added it into the helper (I find myself needing it when writing tests)